pwndbg/commands/heap.py

```diff
@@ -96,6 +97,24 @@ def arena(addr=None):
 
     print(main_arena)
 
+
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+def arenas():
+
+    heap  = pwndbg.heap.current
+    addr  = None
+    arena = heap.get_arena(addr)
+    main_arena_addr = int(arena.address)
+    while addr != main_arena_addr:
+        
+        low,top = heap.get_bounds(addr)
+        print("[%s] begin: %#x end: %#x"%( hex(addr) if addr else 'main',int(low),int(top)))        
```

Comment:
If `get_bounds` returns a `Page` object, you can just print the object (or use `%s`) and everything else is magic